utils.js

```diff
@@ -0,0 +1,30 @@
+export default {
+   /**
+    * helper for scrolling in a grid
+    * @function gridScroll
+    * @param {string} id webix id of the grid
+    * @param {int} h horizontal scroll in pixels
+    * @param {int=0} v veritcal scroll in pixels
+    */
+   gridScroll: async (id, h, v = 0) => {
+      return new Promise((resolve) => {
+         cy.window().then((win) => {
+            console.log(win.AB);
+            win.$$(id).scrollTo(h, v);
+            resolve();
+         });
+      });
+   },
+
+   /**
+    * get ABFactory instance
+    * @function getAB
+    */
+   getAB: async () => {
+      return new Promise((resolve) => {
+         cy.window().then((win) => {
+            resolve(win.AB);
```

Comment:
I'm not sure about using `window.AB`. I have the same question for Johnny (see https://github.com/digi-serve/ab_runtime/pull/363#discussion_r1493953078)

Reply:
Honestly, I don't like "polluting" the global name space like this.  

However, I can't think of another way to make it available to our tests without doing that, so as long as we keep it to a minimum, I'm thinking of keeping this.